src/braket/jobs/metrics_data/cwl_insights_metrics_fetcher.py

```diff
@@ -166,11 +169,15 @@ def get_metrics_for_job(
         query_end_time = job_end_time or int(time.time())
         query_start_time = job_start_time or query_end_time - self.QUERY_DEFAULT_JOB_DURATION
 
-        # The hybrid job name needs to be unique to prevent jobs with similar names from being
-        # conflated.
+        stream_prefix = (
```

Comment:
It kind of sucks that we have to duplicate this logic. Should we just pass in the stream prefix?

Reply:
Perhaps, but this is technically a public interface, so I wanted to avoid breaking it

Reply:
I don't understand - if we're not breaking it by adding arn, why would we be breaking it by adding the stream prefix?

Reply:
Oh I misunderstood. I thought you meant replace the `job_name` parameter with `stream_prefix`, which would be the most direct way of updating the code, but would be a breaking change. We could add an optional `stream_prefix` parameter at the end instead of arn to avoid duplicating logic. I like that